lib/js/atoms/Stepper.types.ts

```diff
@@ -0,0 +1,7 @@
+export interface Step {
+	label: string;
+	iconKey: string;
```

Comment:
Passing `iconKey` can cause some trubble here...
You need to hardcode string, or hack the map of icons to get the key. What do you think about passing icon object instead? Link in `Button` component?